test/test_https.py

```diff
@@ -0,0 +1,31 @@
+from distutils.command.install import install
+import importlib
+import pytest
+import os
+from typing import Dict
+import serial
+import typer
+from typer.testing import CliRunner
+jw = importlib.import_module('init-odkx-sync-endpoint')
+
+env = {"HTTPS_DOMAIN":"odk-x", "HTTPS_ADMIN_EMAIL":"jesse@odk-x.com"}
+
+runner = CliRunner()
+ser = serial.Serial() 
+
+def test_run_cache_setup(mocker):
+    mocker.patch('init-odkx-sync-endpoint.is_enforce_https', return_value=True)
+    assert jw.is_enforce_https() == True
+    result = runner.invoke(jw.app, input="odk-x\n\n\njesse@odk-x.com\n\n")
```

Comment:
@matt9j @wbrunette
is there a way we can pass a an escape sequence for `Control` + `C`?
I really don't what to set up any certificates while running my tests.

Reply:
i will enable pytest execution in our workflow after i am well convict that the testing logic works perfectly.